vendors/cypress/WICED_SDK/apps/demo/aws_demo/aws_demo.mk

```diff
@@ -104,6 +104,7 @@ $(NAME)_SOURCES    := $(AMAZON_FREERTOS_PATH)vendors/cypress/boards/$(PLATFORM)/
                       $(AFR_ABSTRACTIONS_PATH)platform/freertos/iot_network_freertos.c \
                       $(AFR_ABSTRACTIONS_PATH)platform/freertos/iot_threads_freertos.c \
                       $(AFR_C_SDK_STANDARD_PATH)common/iot_init.c \
+					  $(AFR_C_SDK_STANDARD_PATH)common/iot_device_metrics.c \
```

Comment:
This is great thank you so much. Could you make sure you use spaces instead of tabs and update this alignment?

Reply:
Please do this by end of day Monday to have your contribution merged with your name on the commit.